src/lib/db/migrate.task.ts

```
@@ -0,0 +1,26 @@
+import type {Task} from '@feltcoop/gro';
+import ley from 'ley';
+import {defaultPostgresOptions} from '$lib/db/postgres.js';
+
+export const task: Task = {
+	summary: 'running new migrations to bring database up to date',
+	run: async () => {
+		const status = await ley.status({
+			dir: 'migrations',
+			driver: 'postgres',
+			config: defaultPostgresOptions as any,
+		});
+
+		console.log('the following migrations will be run: ', status);
+
+		const successes = await ley.up({
+			dir: 'migrations',
+			driver: 'postgres',
+			config: defaultPostgresOptions as any,
+		});
+		console.log('the following migrations were successful:', successes);
+		if (successes.length != status.length) {
+			console.error('not all pending migrations were applied, please double check');
```

I think this should throw instead of logging since it seems like a problem:

```
throw Error('not all pending migrations were applied, please double check');
```